pdataviewer/src/environments/environment.prod.ts

```diff
@@ -0,0 +1,4 @@
+export const environment = {
```

Comment:
Could you give me a pointer how this is build? 

e.g. yarn install [flag?]

Have you tested this already for the production URL? (-> tested wether the request goes to the correct URL, response should be 404 since we have not started the service yet)

Reply:
We should adjust & test this in ci so that ng is being used instead of npm:

- [ ] Adapt & test docker-image.yaml so it uses ng build (default non-production) instead of npm

Reply:
I tried to implement it just now, you can find it in actions.

Reply:
nice, have you tested it for the latest container:

https://github.com/SCAI-BIO/PDataViewer/pkgs/container/pdataviewer%2Fbackend

Reply:
Yeah it seems to be working for me.